src/NuGet.Core/NuGet.Packaging/Rules/AnalysisResources.resx

```diff
@@ -132,6 +132,10 @@
   <data name="InvalidFrameworkWarning" xml:space="preserve">
```

Comment:
Is this `InvalidFrameworkWarning` still being used anywhere else? if not, then delete the resource as well.

Reply:
It is still used [here](https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.Packaging/Rules/RuleSet.cs#L14).